spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/ApplicationsMetrics.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.controller.support;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Support domain class to map metrics response from a collector.
+ *
+ * @author Janne Valkealahti
+ *
+ */
+public class ApplicationsMetrics {
+
+	private String name;
+	private List<Application> applications;
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public List<Application> getApplications() {
+		return applications;
+	}
+
+	public void setApplications(List<Application> applications) {
+		this.applications = applications;
+	}
+
+	public static class Application {
+
+		private String name;
+		private List<Instance> instances;
+		private double incomingRate;
```

Comment:
How does this `incomingRate` different from the `incomingRate` from the `Instance`. Same is for the `outgoingRate`.